### PR TITLE
Enable processing of cosicorr pixel offset data

### PR DESCRIPTION
**Pull request by forrestfwilliams**
**Description of proposed changes**
References Issue #625 
Adds functionality that allows for the loading of cosicorr pixel offset data.

- adds the prep_cosicorr.py file to generate .rsc files
- makes small changes to load_data.py and readfile.read to allow for using cosicorr as a processor
- in ifgram_inversion, disables conversion from pixel to meter for cosicorr offset data, since it is in meters natively

While prep_cosicorr.py is designed for cosicorr files, it will take any geotiff as input, as long as it is accompanied by a metadata file of the form:

```
fname1.tif date1 date2
fname2.tif date1 date2
...
```
**Reminders**

- [x] Pass Codacy code review (green)
- [x] Pass Circle CI test (green)
- [x] Make sure that your code follows our style. Use the other functions/files as a basis.
- [x] If modifying functionality, describe changes to function behavior and arguments in a comment below the function declaration.
- [x] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Comment by forrestfwilliams** [comment]
@yunjunz let me know if you would like me to send you example data!

File: mintpy/prep_cosicorr.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+from datetime import datetime
+import numpy as np
+from mintpy.utils import readfile, writefile, utils as ut
+
+
+#########################################################################
+EXAMPLE = """example:
+  prep_cosicorr.py  offsets/*offset.tif -m metadata.txt
+  prep_cosicorr.py  snr/*snr.tif -m metadata.txt
+"""
```
Comment (yunjunz) [inline mintpy/prep_cosicorr.py:21]: I would put an example metadata.txt file here for easy reference.

**Comment by yunjunz** [comment]
Thank you @forrestfwilliams, this is awesome! I only have the minor comments above. This PR is good to go in my opinion.

File: mintpy/prep_cosicorr.py (hunk 1 above)
Comment (forrestfwilliams) [reply mintpy/prep_cosicorr.py:21]: Thanks, I'll add this

**Review by yunjunz — APPROVED** [review]

**Pull request merged**